src/main/java/com/codessquad/qna/service/AnswerService.java

```diff
@@ -0,0 +1,27 @@
+package com.codessquad.qna.service;
+
+import com.codessquad.qna.domain.Answer;
+import com.codessquad.qna.domain.Question;
+import com.codessquad.qna.domain.User;
+import com.codessquad.qna.repository.AnswerRepository;
+import com.codessquad.qna.valid.UserValidation;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AnswerService {
+
+    private final AnswerRepository answerRepository;
+    private final QuestionService questionService;
+
+    public AnswerService(AnswerRepository answerRepository, QuestionService questionService) {
+        this.answerRepository = answerRepository;
+        this.questionService = questionService;
+    }
+
+    public void write(User writer, String contents, Long questionId) {
+        UserValidation.validUserInfo(writer);
```

Comment:
```suggestion
        UserValidator.validUserInfo(writer);
```

이렇게 행위자의 느낌으로 작명하는게 더 좋은 것 같아요.

Reply:
행위자로 작명 기억하겠습니다!